File: MyCourses - Android/app/src/main/java/com/mycoursesapp/model/Center.java
package com.mycoursesapp.model;

import com.google.android.gms.maps.model.LatLng;
import com.google.maps.android.clustering.ClusterItem;

/**
 * Created by mamdouhelnakeeb on 2/9/18.
 */

public class Center implements ClusterItem {

    public int id;
    public String name;
//    public String email;
    public String info;
    public String address;
    public double latitude;
    public double longitude;
    public String img;

    public Center(int id, String name, String info, String address, double latitude, double longitude, String img){

        this.id = id;
        this.name = name;
//        this.email = email;
        this.info = info;
        this.address = address;
        this.latitude = latitude;
        this.longitude = longitude;
        this.img = img;
    }

    public Center(){

    }

    @Override
    public LatLng getPosition() {
        return new LatLng(latitude, longitude);
    }
}

File: MyCourses - iOS/My Courses/Models/Center.swift
//
//  Center.swift
//  My Courses
//
//  Created by Mamdouh El Nakeeb on 3/12/18.
//  Copyright © 2018 My Courses. All rights reserved.
//

import Foundation
import UIKit
import GoogleMaps

class Center: NSObject /*,GMUClusterItem*/{
    
    var id = 0
    var name = ""
    var info = ""
    var address = ""
    var latitude = 0.0
    var longitude = 0.0
    var img = ""
    var position = CLLocationCoordinate2D()
    
    init(id: Int, name: String, info: String, address: String, latitude: Double, longitude: Double, img: String) {
        
        self.id = id
        self.name = name
        self.info = info
        self.address = address
        self.latitude = latitude
        self.longitude = longitude
        self.img = img
        
    }

//    init(position: CLLocationCoordinate2D, name: String) {
//        self.position = position
//        self.name = name
//    }
    
    override init() {}
    
}
